RadioDns-PlugIt/SPI/utils.py

```diff
@@ -288,7 +264,7 @@ def extract_pi_data_from_path(path, date):
     return None
 
 
-def generate_pi_file(date, path=None, station=None):
+def generate_pi_file(date, template_uri, path=None, station=None):
```

Comment:
template_uri is not described in the docstring.

Reply:
have added the docstring